### PR TITLE
pl translation

### PR DESCRIPTION
**Pull request by majormarcin**
Polish Translation and config EnableSocietyOwnedVehicles description

### CODE REVIEW

File: config.lua [hunk 1]
```diff
@@ -7,7 +7,7 @@ Config.EnablePlayerManagement     = false
 Config.EnableArmoryManagement     = false
 Config.EnableESXIdentity          = false -- only turn this on if you are using esx_identity
 Config.EnableNonFreemodePeds      = false -- turn this on if you want custom peds
-Config.EnableSocietyOwnedVehicles = false
+Config.EnableSocietyOwnedVehicles = false -- allows use a police garage
```
Comment (ElPumpo) [inline config.lua]: misleading

File: locales/pl.lua [hunk 2]
```diff
@@ -0,0 +1,117 @@
+Locales['en'] = {
```
Comment (ElPumpo) [inline locales/pl.lua]: 1. Change to `pl` or whatever.
2. All translations start with lowercase, exceptions when the first letter isn't in the alfabet

Comment (majormarcin) [reply locales/pl.lua]: done

**Comment by majormarcin** [comment]
fixed

**Comment by ElPumpo** [comment]
Thanks for your help!

**Pull request merged**